assets/js/components/UI/Form/BatchEDTFDate.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useFieldArray } from "react-hook-form";
+import { useFormContext } from "react-hook-form";
+import UIFormFieldArrayAddButton from "@js/components/UI/Form/FieldArrayAddButton";
+import { isEDTFValid } from "../../../services/helpers";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Button } from "@nulib/admin-react-components";
+
+const UIFormBatchEDTFDate = ({
+  name,
+  label,
+  type = "text",
+  required,
+  ...passedInProps
```

Comment:
How about from now on, since this seems to be a more frequent pattern, we always title JS "rest" props, `restProps`, instead of `passedInProps`?